installing_on_kubernetes/_topics/migrate_from_appliance.md

```diff
@@ -0,0 +1,94 @@
+## Preparing for deployment
+
+1. Deploy the CRD (if it doesn't already exist)
+`$ oc get crds | grep manageiqs.manageiq.org`
+`$ oc create -f deploy/crds/manageiq.org_manageiqs_crd.yaml`
+
+2. Set up RBAC
+```bash
+$ oc create -f deploy/role.yaml
+$ oc create -f deploy/role_binding.yaml
+$ oc create -f deploy/service_account.yaml
+```
+
+3. Deploy the operator
+`$ oc create -f deploy/operator.yaml`
```

Comment:
Again I suggest having the example commands placed under the text by using the **```bash**    header.